plugins/gozero/rest/server_middleware_intercepter.go

```diff
@@ -0,0 +1,64 @@
+package rest
+
+import (
+	"fmt"
+	"github.com/apache/skywalking-go/plugins/core/operator"
+	"github.com/apache/skywalking-go/plugins/core/tracing"
+	"github.com/zeromicro/go-zero/rest"
+	"net/http"
+)
+
+type ServerMiddlewareInterceptor struct {
+}
+
+// skyWalking middleware
```

Comment:
```suggestion
// SkyWalking middleware
```

Please keep the project name correctly.